pkg/resources/pullSecret.go

```diff
@@ -33,8 +33,8 @@ func GetDefaultPullSecret(inst *integreatlyv1alpha1.Installation, client k8sclie
 }
 
 // Copys the default pull secret to a target namespace
-func CopyDefaultPullSecretToNameSpace(nameSpaceToCopy string, nameOfSecret string, inst *integreatlyv1alpha1.Installation, client k8sclient.Client, context context.Context) error {
-	openshiftSecret, err := GetDefaultPullSecret(inst, client, context)
+func CopyDefaultPullSecretToNameSpace(context context.Context, nameSpaceToCopy string, inst *integreatlyv1alpha1.Installation, client k8sclient.Client, nameOfSecret string) error {
```

Comment:
For some reason now `nameOfSecret` got moved to the end.

```suggestion
func CopyDefaultPullSecretToNameSpace(context context.Context, nameSpaceToCopy string, nameOfSecret string, inst *integreatlyv1alpha1.Installation, client k8sclient.Client) error {
```

The change on line 20 above also looks to have the `inst` variable moved to the end